ADLA: Update max concurrent jobs & max. parallelism limits in azure-data-lake-analytics-limits.md

arindamc opened the pull request:
Need to update the max concurrent jobs & max parallelism limits to the correct value of 20 & 250 respectively.

azurecla commented:
Hi __@alex-frankel__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

azurecla commented:
__@alex-frankel__, Thanks for signing the contribution license agreement so quickly! Actual humans will now validate the agreement and then evaluate the PR.
<br />Thanks, AZPRBOT;

arindamc committed:
Update azure-data-lake-analytics-limits.md

Need to update the max concurrent jobs & max parallelism limits to the correct value of 20 & 250 respectively.
